observability/otel/otel-collector/otel-collector-receivers.adoc

```diff
@@ -864,6 +864,95 @@ rules:
 <1> The service account of the Collector that has the required ClusterRole `otel-collector` RBAC.
 <2> The list of namespaces to collect events from. The default value is empty, which means that all namespaces are collected.
 
+[id="aws-xray-exporter_{context}"]
+== AWS X-Ray Exporter
+
+The AWS X-Ray exporter converts OpenTelemetry spans to AWS X-Ray Segment Documents and then sends them directly to X-Ray using the PutTraceSegments API.
+
+:FeatureName: The AWS X-Ray exporter
+include::snippets/technology-preview.adoc[]
+
+.OpenTelemetry Collector custom resource with the enabled AWS X-Ray Exporter
+[source,yaml]
+----
+# ...
+  config:
+    exporters:
+      awsxray:
+        region: eu-west-1 # <1>
```

Comment:
@pavolloffay, no double quotes for `eu-west-1` here?

Reply:
you can add them for consistency, but it should not matter